src/core/model.jl

```diff
@@ -64,17 +65,27 @@ which are the fields of `AgentBasedModel`.
 `scheduler = fastest` decides the order with which agents are activated
 (see e.g. [`by_id`](@ref) and the scheduler API).
 
+`rng = Random.default_rng()` provides random number generation to the model.
+Passing, for example `MersenneTwister(1234)` will initialise with a repeatable random
+seed, and `RandomDevice()` will use the system's entropy source (coupled with hardware
+like [TrueRNG](https://ubld.it/truerng_v3) will invoke a true random source, rather
+than pseudo-random methods like `MersenneTwister`).
```

Comment:
I think that is not necessary information. We can simply say that it must be a subtype of `AbstractRNG`. This information can be in the Tutorial.md documentation page instead. 

(I believe docstrings should be as lean as humanly possible)